solutions/standard/variables.tf

```diff
@@ -0,0 +1,310 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud platform API key needed to deploy IAM enabled resources."
+  sensitive   = true
+}
+
+variable "region" {
+  type        = string
+  description = "The region where the resources are created."
```

Comment:
This would have to match to VPC region, correct? Suggest to include that in descrption

Reply:
actually, can't the VPC region be got from the `existing_vpc_crn` input?